Userland/Utilities/icc.cpp

```diff
@@ -30,12 +30,44 @@ static void out_optional(char const* label, Optional<T> const& optional)
         outln("(not set)");
 }
 
+static void out_parametric_curve(Gfx::ICC::ParametricCurveTagData const& parametric_curve, int indent_amount)
+{
+    auto indent = MUST(String::repeated(' ', indent_amount));
+    switch (parametric_curve.function_type()) {
+    case Gfx::ICC::ParametricCurveTagData::FunctionType::Type0:
+        outln("{}Y = X**{}", indent, parametric_curve.g());
+        break;
+    case Gfx::ICC::ParametricCurveTagData::FunctionType::Type1:
+        outln("{}Y = ({}*X + {})**{}   if X >= -{}/{}", indent,
+            parametric_curve.a(), parametric_curve.b(), parametric_curve.g(), parametric_curve.b(), parametric_curve.a());
+        outln("{}Y = 0                                else", indent);
+        break;
+    case Gfx::ICC::ParametricCurveTagData::FunctionType::Type2:
+        outln("{}Y = ({}*X + {})**{} + {}   if X >= -{}/{}", indent,
+            parametric_curve.a(), parametric_curve.b(), parametric_curve.g(), parametric_curve.c(), parametric_curve.b(), parametric_curve.a());
+        outln("{}Y =  {}                                    else", indent, parametric_curve.c());
+        break;
+    case Gfx::ICC::ParametricCurveTagData::FunctionType::Type3:
+        outln("{}Y = ({}*X + {})**{}   if X >= {}", indent,
+            parametric_curve.a(), parametric_curve.b(), parametric_curve.g(), parametric_curve.d());
+        outln("{}Y =  {}*X                         else", indent, parametric_curve.c());
+        break;
+    case Gfx::ICC::ParametricCurveTagData::FunctionType::Type4:
+        outln("{}Y = ({}*X + {})**{} + {}   if X >= {}", indent,
+            parametric_curve.a(), parametric_curve.b(), parametric_curve.g(), parametric_curve.e(), parametric_curve.d());
+        outln("{}Y =  {}*X + {}                             else", indent, parametric_curve.c(), parametric_curve.f());
+        break;
+    }
+}
+
 static void out_curves(Vector<Gfx::ICC::LutCurveType> const& curves)
 {
     auto indent = MUST(String::repeated(' ', 8));
-    for (auto const& curve: curves) {
+    for (auto const& curve : curves) {
```

Comment:
Style fix belongs in the previous commit please :)

Reply:
Whoops sorry, how very careless. Done.